package.json

```diff
@@ -1,47 +1,71 @@
 {
-  "name": "awesome-react-typescript-boilerplate",
-  "version": "1.0.0",
-  "description": "",
-  "main": "index.js",
-  "scripts": {
-    "start": "webpack-dev-server --config webpack/webpack.dev.js",
-    "build": "webpack --config webpack/webpack.prod.js"
-  },
-  "keywords": [],
-  "author": "",
-  "license": "ISC",
-  "devDependencies": {
-    "@babel/core": "^7.4.0",
-    "@babel/plugin-proposal-class-properties": "7.4.0",
-    "@babel/plugin-proposal-object-rest-spread": "7.4.3",
-    "@babel/plugin-syntax-dynamic-import": "7.2.0",
-    "@babel/plugin-transform-react-constant-elements": "7.2.0",
-    "@babel/plugin-transform-react-inline-elements": "7.2.0",
-    "@babel/preset-env": "7.4.3",
-    "@babel/preset-react": "7.0.0",
-    "@babel/preset-typescript": "7.3.3",
-    "@types/react": "16.8.10",
-    "@types/react-dom": "16.8.3",
-    "babel-loader": "8.0.5",
-    "babel-plugin-lodash": "3.3.4",
-    "babel-plugin-transform-react-remove-prop-types": "0.4.24",
-    "clean-webpack-plugin": "2.0.1",
-    "css-loader": "2.1.1",
-    "fork-ts-checker-webpack-plugin": "1.0.1",
-    "html-webpack-plugin": "3.2.0",
-    "mini-css-extract-plugin": "0.5.0",
-    "node-sass": "4.11.0",
-    "optimize-css-assets-webpack-plugin": "5.0.1",
-    "sass-loader": "7.1.0",
-    "style-loader": "0.23.1",
-    "typescript": "3.4.1",
-    "webpack": "4.29.6",
-    "webpack-cli": "3.3.0",
-    "webpack-dev-server": "3.2.1",
-    "webpack-merge": "4.2.1"
-  },
-  "dependencies": {
-    "react": "16.8.6",
-    "react-dom": "16.8.6"
-  }
+	"name": "awesome-react-typescript-boilerplate",
+	"version": "1.0.0",
+	"description": "",
+	"main": "index.js",
+	"scripts": {
+		"start": "webpack-dev-server --config webpack/webpack.dev.js",
+		"build": "webpack --config webpack/webpack.prod.js",
+		"lint": "tslint-config-prettier-check ./tslint.json",
+		"check-types": "tsc",
+		"format": "prettier \"**/*.+(js|jsx|json|ts|tsx|css|scss|md|mdx|graphql)\" --write"
+	},
+	"husky": {
+		"hooks": {
+			"pre-commit": "lint-staged"
+		}
+	},
+	"lint-staged": {
+		"src/*.(ts, tsx)": "tslint",
```

Comment:
Wrong format `src/*.(ts, tsx)` would not match anything. Use the `src/*.{ts,tsx}` instead